Reflect.ts

```diff
@@ -1714,8 +1724,8 @@ namespace Reflect {
                 delete(value: T): boolean { return this._map.delete(value); }
                 clear(): void { this._map.clear(); }
                 keys() { return this._map.keys(); }
-                values() { return this._map.values(); }
-                entries() { return this._map.entries(); }
+                values() { return this._map.keys(); }
+                entries() { return this._map.keys(); }

```

Comment:
`keys` iterator on `entries`?O_o

Reply:
It isn't actually used, so it didn't matter much.

Reply:
Fixed in #146